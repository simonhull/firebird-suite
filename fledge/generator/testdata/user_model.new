package models

import (
	"database/sql"
	"time"
)

// User represents a user in the system
type User struct {
	ID        int64     `json:"id"`
	Email     string    `json:"email"`
	FirstName string    `json:"first_name"`
	LastName  string    `json:"last_name"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}