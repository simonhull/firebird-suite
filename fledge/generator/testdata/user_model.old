package models

import (
	"database/sql"
	"time"
)

// User represents a user in the system
type User struct {
	ID        int64     `json:"id"`
	Email     string    `json:"email"`
	Name      string    `json:"name"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}