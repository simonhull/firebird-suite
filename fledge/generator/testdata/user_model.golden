--- old.txt
+++ new.txt
@@ -9,7 +9,8 @@
 type User struct {
     ID        int64     `json:"id"`
     Email     string    `json:"email"`
-    Name      string    `json:"name"`
+    FirstName string    `json:"first_name"`
+    LastName  string    `json:"last_name"`
     CreatedAt time.Time `json:"created_at"`
     UpdatedAt time.Time `json:"updated_at"`
 }
